docs/meshstack.kubernetes.index.md

```diff
@@ -3,16 +3,18 @@ id: meshstack.kubernetes.index
 title: Integration
 ---
 
+
 meshStack supports project creation, configuration, user management and SSO for Kubernetes clusters.
 Our Kubernetes integration is distribution-independent and is purely based on "vanilla" upstream Kubernetes services.
 Operators should be able to successfully integrate any Kubernetes distribution of their chosing as long as it has the
 required APIs and configuration options available.
+We also support Azure AAD authenticated users to get seamless access to the AKS (Azure Kubernetes) cluster.
 
 ## Integration Overview
 
 To enable integration with Kubernetes, operators deploy and configure the meshStack Kubernetes Module. Operators can configure one or multiple `PlatformInstance`s of `PlatformType` Kubernetes. This makes Kubernetes available to meshProjects like any other cloud platform in meshStack.
 
-meshStack automatically configures Kubernetes namespaces and RBAC permissions to integrate SSO with [meshStack Identity Federation](./meshstack.identity-federation.md).
+Depending on which Kubernetes distribution you want to use you will need to follow different steps. Below are the steps for the [meshStack Identity Federation](./meshstack.identity-federation.md) integration as well as for the Azure Kubernetes Services.
```

Comment:
d: Can you also link to the Azure Kubernetes Services section here?

Reply:
sure